tests/test_extra.lua

```diff
@@ -905,6 +905,111 @@ T['pickers']['buf_lines()']['validates arguments'] = function()
   validate({ scope = '1' }, '`pickers%.buf_lines`.*"scope".*"1".*one of')
 end
 
+T['pickers']['colorschemes()'] = new_set()
+
+local pick_colorschemes = forward_lua_notify('MiniExtra.pickers.colorschemes')
+
+T['pickers']['colorschemes()']['works'] = function()
+  child.set_size(10, 80)
+  child.cmd('colorscheme default')
+
+  child.lua_notify('_G.return_item = MiniExtra.pickers.colorschemes()')
+  validate_picker_name('Colorschemes')
+  type_keys('ha')
```

Comment:
Let's test it against something that is owned by 'mini.nvim'. Typing `mini` should be more robust.

Reply:
Sure. I'll change all of them later tonight.

Reply:
A tip: to regenerate a screenshot without having to delete files manually, pass `force = true` to `child.expect_screenshot()`.